web/styles/popovers.css

```diff
@@ -738,11 +738,33 @@ ul {
     left: -1px;
 }
 
-.giphy-popover {
+.giphy-popover,
+.emoji-popover-root {
     .tippy-content {
         /* We remove the default padding from this container
            as it is not necessary for the Giphy popover. */
         padding: 0;
+
+        /* By resetting to the default color from the `body`,
+           we can ignore the colors applied from `tippy-box`. */
+        color: var(--color-text-default);
+    }
+}
+
+.emoji-popover-root {
+    /* Fixes incorrect emoji popover rendering on small screens in edit mode.  */
+    [data-reference-hidden] {
+        display: inherit;
+    }
+
+    /* We consider that the arrow of the emoji popover could be
+       either at the top or bottom, not on the left or right. */
+    .tippy-box[data-placement="top"] .tippy-arrow::before {
+        border-top-color: hsl(0deg 0% 93%);
+    }
+
+    .tippy-box[data-placement="bottom"] .tippy-arrow::before {
+        border-bottom-color: hsl(0deg 0% 93%);
```

Comment:
The "we consider" comment is confusing to me -- what happens for left/right placement? If the claim is that this will never have left/right placement, we should say that, not just that this code only considers the other possibilities. 

And what is the purpose of having this custom CSS for just this popover, and not others? Generally we want them to have similar styling, so some explanation of what this is for would be appropriate.

Reply:
This comment is wrong. We don't limit where this popover can be placed. The main point here is that the emoji popover has different background color for the header and footer, so we change the arrow to match these colors.

Reply:
I've fixed the comment.